test/facade_converter_test.ts

```diff
@@ -35,9 +35,19 @@ function getSources(str: string): {[k: string]: string} {
     'angular2/typings/es6-shim/es6-shim': es6RuntimeDeclarations,
     'angular2/src/core/di/forward_ref.d.ts': `
         export declare function forwardRef<T>(x: T): T;`,
-    'angular2/typings/es6-promise/es6-promise.d.ts': `
-        declare class Promise<R> {}
-        declare module Promise {}`,
+    'typings/es6-promise/es6-promise.d.ts': `
```

Comment:
doesn't have to be in this change, but it's clear we should just fetch typings in the postinstall for this project, and pull the content from it, instead of copying it in here